src/components/HashtagTool/TrendingSideBar.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+
+function TrendingSidebar() {
+    const suggestions = [
+        'meme',
+        'nostr',
+        'grownstr',
+        'bitcoin',
+        'plebchain',
+        'siamstr',
+        'btc',
+        'privacy',
+        'security',
+        'memes',
+        'tunestr',
+        'music',
+        'coffeechain',
+        'press',
+        'presse',
+        'memestr',
+        'photography',
+        'funny',
+        'france',
+        'artstr',
+        'foodstr',
+        'news',
+        'btcprague',
+        'sats',
+        'zaps',
+        'china',
+        'permaculture',
+        'inflation',
+        'primal',
+        'plebs',
+        'permies',
+        'photestr',
+        'homesteading',
+        'bible',
+        'biblestr',
+        'christian',
+        'crypto',
+        'yestr',
+        'new',
+        'thainostrich',
+        'nostrich',
+        'wisdom',
+        'hodl',
+        'us',
+        'dance',
+    ];
+    const navigate = useNavigate();
+    const handleTagClick = suggestions => {
+        navigate(`/search/${suggestions}`);
+    };
+
+    return (
+        <aside className="hidden lg:block w-1/4 bg-white p-2 sticky top-0 h-screen overflow-y-auto border-l border-gray-200 z-40 ">
+            <h2 className="text-2xl font-bold text-center pt-14 text-gray-900 mb-4 ">
```

Comment:
the pt-14 looks very weird once the header bar is hidden. It makes it look really out of place.
Either make it responsive by attaching it with header bar so that when header bar appears, it shifts down and when header bar disappears, it shifts up.
OR
detach it from top of screen and it it a little lower so it looks like it is intentionally placed there. 
One similar idea is to have a side section be completely seperate from header bar and make the header bar operate only in the feed area. Just like we have a section on the left side for category, we can have another section on the right for the widgets. Something like primal.net does on their website. What are your thoughts on this?